packages/dm-core-plugins/src/table/TableRow/TableRow.tsx

```diff
@@ -178,7 +175,11 @@ export function TableRow(props: TTableRow) {
           <Table.Cell colSpan={getColumnsLength()}>
             <ViewCreator
               idReference={`${idReference}[${item.index}]`}
-              viewConfig={config.views?.[item.index] ?? config.defaultView}
+              viewConfig={
+                config.expandableRecipeViewConfig
+                  ? config.expandableRecipeViewConfig
+                  : config.views?.[item.index] ?? config.defaultView
```

Comment:
I wonder if the config.views and config.defaultView makes sense anymore. I see they are defined in TTablePluginConfig, but not in TablePluginConfig.json. They don't seem to be used anywhere in our example either. What do you think @eoaksnes?

Reply:
```suggestion
                  : { type: 'ViewConfig', scope: 'self' }
```